.gitignore

```diff
@@ -58,3 +58,6 @@ target/
 
 # DS_Store
 *.DS_Store
+
+# pycharm
```

Comment:
@seb-elico since we have many developer using pycharm , it's necessary to add this in all the repo.